src/app/participant-list/participant-list.component.ts

```diff
@@ -1038,6 +1038,10 @@ export class ParticipantListComponent implements OnInit {
         this.selectedMR = "";
         this.selectedOncOrTissue = "";
       }
+      if (participant.data.activities !== null){
+        let p = participant.participantData.find( p => p.data["COLLABORATOR_PARTICIPANT_ID"].slice(-2)=== "_3")
```

Comment:
maybe you could use `p.data['MEMBER_TYPE'] === 'SELF'` would be more convenient?

Reply:
I totally agree, the problem is ticket said ```, please default to the individual that is designated as proband or has an _3 in their name.```